### ISSUE TITLE
[BUG] Cooldown reporting unix time and waiting ungodly numbers (Not banned issue)

### ISSUE THREAD

**Issue by ZorudaRinku** [rdeepak2002/reddit-place-script-2022#101]
# Bug Report Form

## Checklist

- [X] I am using the latest version of the code.
- [X] I am absolutely sure I read the README in its entirety.
- [X] I have made sure this is not a duplicate issue.
- [X] I have filled out this form correctly.

## Describe the bug

When using the bot, it will use all workers' availability then get stuck in a big loop stating that the next pixel will be placed in ~1648928310 seconds. Seems like this is unix timestamp and it's waiting for the entire duration.

## To Reproduce

Steps to trigger the bug:

Fill out your workers
Start the bot
Let all workers place at least 1 pixel

## Expected behavior

Wait the correct amount of time then place another pixel for each worker

## Additional context

Console gets spammed with this text:
```[02-Apr-22 13:41:29] :: [INFO] - Thread #31 :: 
[02-Apr-22 13:41:29] :: [INFO] - Thread #0 :: 
[02-Apr-22 13:41:29] :: [INFO] - Thread #19 :: 
[02-Apr-22 13:41:29] :: [INFO] - Thread #32 :: 
[02-Apr-22 13:41:29] :: [INFO] - Thread #11 :: 
[02-Apr-22 13:41:29] :: [INFO] - Thread #10 :: 
[02-Apr-22 13:41:29] :: [INFO] - Thread #7 :: 1648928310.0 seconds until next pixel is drawn
[02-Apr-22 13:41:29] :: [INFO] - Thread #9 :: 
[02-Apr-22 13:41:30] :: [INFO] - Thread #13 ::```

**Comment by nbourre** [comment]
That's an easy fix that works. We're not launching a rocket.

**Comment by ZorudaRinku** [comment]
As I posted this I noticed that easy fix actually makes the threads display negative time and continue waiting. So I guess I'll edit that out of the report

**Comment by ZorudaRinku** [comment]
Bug fixed in merged PR #110
